### PR TITLE
Updated docs for pathname

### LINKED ISSUE #9412: Unexpected pathname value returned from useRouter
# Bug report

## Describe the bug

The `pathname` returned from `useRouter` doesn't match the description of this data in the [readme](https://github.com/zeit/next.js#routing).

I'm unsure if this is a bug, or simply incomplete information in the current docs. Regardless, this is a point of semi-regular confusion on our team.

## To Reproduce

1. Create the following page:
```tsx
[locale]/posts/[id].tsx
```

2. Within this page, use `useRouter`:

```tsx
import { useRouter } from 'next/router'

const Page = () => {
  const router = useRouter()
  console.log(router)

  return (<h1>Example</h1>)
}

export default Page
```

3. Visit this page in the browser, with some query string (i.e. `/fr/posts/abc?q=123`)
4. View the log (either server logs, or in the browser console):
```ts
{
  route: "/[locale]/posts/[id]",
  pathname: "/[locale]/posts/[id]",
  query: {
    locale: "fr",
    id: "abc",
    "q": "123"
  },
  asPath: "/fr/posts/abc?q=123"
}
```

## Expected behavior

Based on the following descriptions of these values:

> - `route` - String of the current route
> - `pathname` - String of the current path excluding the query string
> - `asPath` - String of the actual path (including the query) shows in the browser
> [Reference](https://github.com/zeit/next.js#routing)

I would expect the following `useRouter` results:

```diff
{
  route: "/[locale]/posts/[id]",
- pathname: "/[locale]/posts/[id]",
+ pathname: "/fr/posts/abc",
  query: {
    locale: "fr",
    id: "abc",
    "q": "123"
  },
  asPath: "/fr/posts/abc?q=123"
}
```

## Screenshots

N/A

## System information

- OS: macOS
- Browser: N/A[e.g. chrome, safari]
- Version of Next.js: `9.0.7`

## Additional context

None

### PR DESCRIPTION
**Pull request by lfades**
Is it `path of the page` or `path to the page`? 🤔 

Closes #9412

### CODE REVIEW

**Comment by ijjk** [comment]
# Stats from current PR

<details>
<summary><strong>Default Server Mode</strong></summary>

<br/>

<details>
<summary><strong>General</strong> Overall increase ⚠️</summary>

|  | zeit/next.js canary  | lfades/next.js docs-update-pathname | Change |
| - | - | - | - |
| buildDuration | 12.5s | 12.6s | ⚠️ +71ms |
| nodeModulesSize | 48.9 MB | 48.9 MB | ⚠️ +64 B |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons)</strong></summary>

|  | zeit/next.js canary  | lfades/next.js docs-update-pathname | Change |
| - | - | - | - |
| main-HASH.js gzip | 5.13 kB | 5.13 kB | ✓ |
| webpack-HASH.js gzip | 746 B | 746 B | ✓ |
| 4952ddcd88e7..54d3.js gzip | 4.68 kB | 4.68 kB | ✓ |
| commons.HASH.js gzip | 4.06 kB | 4.06 kB | ✓ |
| de003c3a9d30..4cf7.js gzip | 13.7 kB | 13.7 kB | ✓ |
| framework.HASH.js gzip | 39.1 kB | 39.1 kB | ✓ |
| Overall change | 67.5 kB | 67.5 kB | ✓ |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons) Modern</strong></summary>

|  | zeit/next.js canary  | lfades/next.js docs-update-pathname | Change |
| - | - | - | - |
| main-HASH.module.js gzip | 4.19 kB | 4.19 kB | ✓ |
| webpack-HASH..dule.js gzip | 746 B | 746 B | ✓ |
| 4952ddcd88e7..dule.js gzip | 5.56 kB | 5.56 kB | ✓ |
| de003c3a9d30..dule.js gzip | 12.5 kB | 12.5 kB | ✓ |
| framework.HA..dule.js gzip | 39.1 kB | 39.1 kB | ✓ |
| Overall change | 62.1 kB | 62.1 kB | ✓ |

</details>

<details>
<summary><strong>Legacy Client Bundles (polyfills)</strong></summary>

|  | zeit/next.js canary  | lfades/next.js docs-update-pathname | Change |
| - | - | - | - |
| polyfills-HASH.js gzip | 4.76 kB | 4.76 kB | ✓ |
| Overall change | 4.76 kB | 4.76 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages</strong></summary>

|  | zeit/next.js canary  | lfades/next.js docs-update-pathname | Change |
| - | - | - | - |
| _app.js gzip | 1.33 kB | 1.33 kB | ✓ |
| _error.js gzip | 4.07 kB | 4.07 kB | ✓ |
| hooks.js gzip | 779 B | 779 B | ✓ |
| index.js gzip | 222 B | 222 B | ✓ |
| link.js gzip | 2.9 kB | 2.9 kB | ✓ |
| routerDirect.js gzip | 283 B | 283 B | ✓ |
| withRouter.js gzip | 282 B | 282 B | ✓ |
| Overall change | 9.87 kB | 9.87 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages Modern</strong></summary>

|  | zeit/next.js canary  | lfades/next.js docs-update-pathname | Change |
| - | - | - | - |
| _app.module.js gzip | 757 B | 757 B | ✓ |
| _error.module.js gzip | 3.06 kB | 3.06 kB | ✓ |
| hooks.module.js gzip | 371 B | 371 B | ✓ |
| index.module.js gzip | 212 B | 212 B | ✓ |
| link.module.js gzip | 2.47 kB | 2.47 kB | ✓ |
| routerDirect..dule.js gzip | 273 B | 273 B | ✓ |
| withRouter.m..dule.js gzip | 272 B | 272 B | ✓ |
| Overall change | 7.41 kB | 7.41 kB | ✓ |

</details>

<details>
<summary><strong>Client Build Manifests</strong></summary>

|  | zeit/next.js canary  | lfades/next.js docs-update-pathname | Change |
| - | - | - | - |
| _buildManifest.js gzip | 61 B | 61 B | ✓ |
| _buildManife..dule.js gzip | 61 B | 61 B | ✓ |
| Overall change | 122 B | 122 B | ✓ |

</details>

<details>
<summary><strong>Rendered Page Sizes</strong></summary>

|  | zeit/next.js canary  | lfades/next.js docs-update-pathname | Change |
| - | - | - | - |
| index.html gzip | 1.02 kB | 1.02 kB | ✓ |
| link.html gzip | 1.03 kB | 1.03 kB | ✓ |
| withRouter.html gzip | 1.02 kB | 1.02 kB | ✓ |
| Overall change | 3.07 kB | 3.07 kB | ✓ |

</details>

</details>
<hr/>
<details>
<summary><strong>Serverless Mode</strong></summary>

<br/>

<details>
<summary><strong>General</strong> Overall increase ⚠️</summary>

|  | zeit/next.js canary  | lfades/next.js docs-update-pathname | Change |
| - | - | - | - |
| buildDuration | 12.7s | 12.5s | -226ms |
| nodeModulesSize | 48.9 MB | 48.9 MB | ⚠️ +64 B |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons)</strong></summary>

|  | zeit/next.js canary  | lfades/next.js docs-update-pathname | Change |
| - | - | - | - |
| main-HASH.js gzip | 5.13 kB | 5.13 kB | ✓ |
| webpack-HASH.js gzip | 746 B | 746 B | ✓ |
| 4952ddcd88e7..54d3.js gzip | 4.68 kB | 4.68 kB | ✓ |
| commons.HASH.js gzip | 4.06 kB | 4.06 kB | ✓ |
| de003c3a9d30..4cf7.js gzip | 13.7 kB | 13.7 kB | ✓ |
| framework.HASH.js gzip | 39.1 kB | 39.1 kB | ✓ |
| Overall change | 67.5 kB | 67.5 kB | ✓ |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons) Modern</strong></summary>

|  | zeit/next.js canary  | lfades/next.js docs-update-pathname | Change |
| - | - | - | - |
| main-HASH.module.js gzip | 4.19 kB | 4.19 kB | ✓ |
| webpack-HASH..dule.js gzip | 746 B | 746 B | ✓ |
| 4952ddcd88e7..dule.js gzip | 5.56 kB | 5.56 kB | ✓ |
| de003c3a9d30..dule.js gzip | 12.5 kB | 12.5 kB | ✓ |
| framework.HA..dule.js gzip | 39.1 kB | 39.1 kB | ✓ |
| Overall change | 62.1 kB | 62.1 kB | ✓ |

</details>

<details>
<summary><strong>Legacy Client Bundles (polyfills)</strong></summary>

|  | zeit/next.js canary  | lfades/next.js docs-update-pathname | Change |
| - | - | - | - |
| polyfills-HASH.js gzip | 4.76 kB | 4.76 kB | ✓ |
| Overall change | 4.76 kB | 4.76 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages</strong></summary>

|  | zeit/next.js canary  | lfades/next.js docs-update-pathname | Change |
| - | - | - | - |
| _app.js gzip | 1.33 kB | 1.33 kB | ✓ |
| _error.js gzip | 4.07 kB | 4.07 kB | ✓ |
| hooks.js gzip | 779 B | 779 B | ✓ |
| index.js gzip | 222 B | 222 B | ✓ |
| link.js gzip | 2.9 kB | 2.9 kB | ✓ |
| routerDirect.js gzip | 283 B | 283 B | ✓ |
| withRouter.js gzip | 282 B | 282 B | ✓ |
| Overall change | 9.87 kB | 9.87 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages Modern</strong></summary>

|  | zeit/next.js canary  | lfades/next.js docs-update-pathname | Change |
| - | - | - | - |
| _app.module.js gzip | 757 B | 757 B | ✓ |
| _error.module.js gzip | 3.06 kB | 3.06 kB | ✓ |
| hooks.module.js gzip | 371 B | 371 B | ✓ |
| index.module.js gzip | 212 B | 212 B | ✓ |
| link.module.js gzip | 2.47 kB | 2.47 kB | ✓ |
| routerDirect..dule.js gzip | 273 B | 273 B | ✓ |
| withRouter.m..dule.js gzip | 272 B | 272 B | ✓ |
| Overall change | 7.41 kB | 7.41 kB | ✓ |

</details>

<details>
<summary><strong>Client Build Manifests</strong></summary>

|  | zeit/next.js canary  | lfades/next.js docs-update-pathname | Change |
| - | - | - | - |
| _buildManifest.js gzip | 61 B | 61 B | ✓ |
| _buildManife..dule.js gzip | 61 B | 61 B | ✓ |
| Overall change | 122 B | 122 B | ✓ |

</details>

<details>
<summary><strong>Serverless bundles</strong></summary>

|  | zeit/next.js canary  | lfades/next.js docs-update-pathname | Change |
| - | - | - | - |
| _error.js gzip | 77.8 kB | 77.8 kB | ✓ |
| hooks.html gzip | 1.05 kB | 1.05 kB | ✓ |
| index.js gzip | 78 kB | 78 kB | ✓ |
| link.js gzip | 80.4 kB | 80.4 kB | ✓ |
| routerDirect.js gzip | 78.1 kB | 78.1 kB | ✓ |
| withRouter.js gzip | 78.1 kB | 78.1 kB | ✓ |
| Overall change | 393 kB | 393 kB | ✓ |

</details>

</details>


Commit: 508962bb78c159cd64751ebcdf969ee5118147a1

File: docs/api-reference/next/router.md [hunk 1]
```diff
@@ -37,8 +37,8 @@ export default ActiveLink
 
 The following is the definition of the `router` object returned by both [`useRouter`](#useRouter) and [`withRouter`](#withRouter):
 
-- `route`: `String` - Current route
-- `pathname`: `String` - Current path excluding the query string
+- `route`: `String` - Current route. That is the path of the page in `/pages`.
```
Comment (timneutkens) [inline docs/api-reference/next/router.md]:
```suggestion
```

File: packages/next/README.md [hunk 2]
```diff
@@ -807,8 +807,8 @@ See [Disabling File-System Routing](#disabling-file-system-routing).
 
 Above `Router` object comes with the following API:
 
-- `route` - `String` of the current route
-- `pathname` - `String` of the current path excluding the query string
+- `route` - `String` of the current route. That is the path of the page in `/pages`.
```
Comment (timneutkens) [inline packages/next/README.md]:
```suggestion
```

File: packages/next/README.md [hunk 3]
```diff
@@ -807,8 +807,8 @@ See [Disabling File-System Routing](#disabling-file-system-routing).
...
+- `pathname` - `String` similar to `route`, but can include a trailing backslash.
```
Comment (timneutkens) [inline packages/next/README.md]:
```suggestion
- `pathname` - `String` of the current route. That is the path of the page in `/pages`.
```

File: docs/api-reference/next/router.md [hunk 4]
```diff
@@ -37,8 +37,8 @@ export default ActiveLink
...
+- `pathname`: `String` - Same as `route`, but can include a trailing backslash.
```
Comment (timneutkens) [inline docs/api-reference/next/router.md]:
```suggestion
- `pathname`: Current route. That is the path of the page in `/pages`.
```

**Comment by ijjk** [comment]
# Stats from current PR

<details>
<summary><strong>Default Server Mode</strong> (Decrease detected ✓)</summary>

<br/>

<details>
<summary><strong>General</strong> Overall decrease ✓</summary>

|  | zeit/next.js canary  | lfades/next.js docs-update-pathname | Change |
| - | - | - | - |
| buildDuration | 12.7s | 12.9s | ⚠️ +205ms |
| nodeModulesSize | 48.9 MB | 48.9 MB | -15 B |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons)</strong></summary>

|  | zeit/next.js canary  | lfades/next.js docs-update-pathname | Change |
| - | - | - | - |
| main-HASH.js gzip | 5.13 kB | 5.13 kB | ✓ |
| webpack-HASH.js gzip | 746 B | 746 B | ✓ |
| 4952ddcd88e7..54d3.js gzip | 4.68 kB | 4.68 kB | ✓ |
| commons.HASH.js gzip | 4.06 kB | 4.06 kB | ✓ |
| de003c3a9d30..4cf7.js gzip | 13.7 kB | 13.7 kB | ✓ |
| framework.HASH.js gzip | 39.1 kB | 39.1 kB | ✓ |
| Overall change | 67.5 kB | 67.5 kB | ✓ |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons) Modern</strong></summary>

|  | zeit/next.js canary  | lfades/next.js docs-update-pathname | Change |
| - | - | - | - |
| main-HASH.module.js gzip | 4.19 kB | 4.19 kB | ✓ |
| webpack-HASH..dule.js gzip | 746 B | 746 B | ✓ |
| 4952ddcd88e7..dule.js gzip | 5.56 kB | 5.56 kB | ✓ |
| de003c3a9d30..dule.js gzip | 12.5 kB | 12.5 kB | ✓ |
| framework.HA..dule.js gzip | 39.1 kB | 39.1 kB | ✓ |
| Overall change | 62.1 kB | 62.1 kB | ✓ |

</details>

<details>
<summary><strong>Legacy Client Bundles (polyfills)</strong></summary>

|  | zeit/next.js canary  | lfades/next.js docs-update-pathname | Change |
| - | - | - | - |
| polyfills-HASH.js gzip | 4.76 kB | 4.76 kB | ✓ |
| Overall change | 4.76 kB | 4.76 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages</strong></summary>

|  | zeit/next.js canary  | lfades/next.js docs-update-pathname | Change |
| - | - | - | - |
| _app.js gzip | 1.33 kB | 1.33 kB | ✓ |
| _error.js gzip | 4.07 kB | 4.07 kB | ✓ |
| hooks.js gzip | 779 B | 779 B | ✓ |
| index.js gzip | 222 B | 222 B | ✓ |
| link.js gzip | 2.9 kB | 2.9 kB | ✓ |
| routerDirect.js gzip | 283 B | 283 B | ✓ |
| withRouter.js gzip | 282 B | 282 B | ✓ |
| Overall change | 9.87 kB | 9.87 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages Modern</strong></summary>

|  | zeit/next.js canary  | lfades/next.js docs-update-pathname | Change |
| - | - | - | - |
| _app.module.js gzip | 757 B | 757 B | ✓ |
| _error.module.js gzip | 3.06 kB | 3.06 kB | ✓ |
| hooks.module.js gzip | 371 B | 371 B | ✓ |
| index.module.js gzip | 212 B | 212 B | ✓ |
| link.module.js gzip | 2.47 kB | 2.47 kB | ✓ |
| routerDirect..dule.js gzip | 273 B | 273 B | ✓ |
| withRouter.m..dule.js gzip | 272 B | 272 B | ✓ |
| Overall change | 7.41 kB | 7.41 kB | ✓ |

</details>

<details>
<summary><strong>Client Build Manifests</strong></summary>

|  | zeit/next.js canary  | lfades/next.js docs-update-pathname | Change |
| - | - | - | - |
| _buildManifest.js gzip | 61 B | 61 B | ✓ |
| _buildManife..dule.js gzip | 61 B | 61 B | ✓ |
| Overall change | 122 B | 122 B | ✓ |

</details>

<details>
<summary><strong>Rendered Page Sizes</strong></summary>

|  | zeit/next.js canary  | lfades/next.js docs-update-pathname | Change |
| - | - | - | - |
| index.html gzip | 1.02 kB | 1.02 kB | ✓ |
| link.html gzip | 1.03 kB | 1.03 kB | ✓ |
| withRouter.html gzip | 1.02 kB | 1.02 kB | ✓ |
| Overall change | 3.07 kB | 3.07 kB | ✓ |

</details>

</details>
<hr/>
<details>
<summary><strong>Serverless Mode</strong> (Decrease detected ✓)</summary>

<br/>

<details>
<summary><strong>General</strong> Overall decrease ✓</summary>

|  | zeit/next.js canary  | lfades/next.js docs-update-pathname | Change |
| - | - | - | - |
| buildDuration | 13.2s | 13.2s | -1ms |
| nodeModulesSize | 48.9 MB | 48.9 MB | -15 B |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons)</strong></summary>

|  | zeit/next.js canary  | lfades/next.js docs-update-pathname | Change |
| - | - | - | - |
| main-HASH.js gzip | 5.13 kB | 5.13 kB | ✓ |
| webpack-HASH.js gzip | 746 B | 746 B | ✓ |
| 4952ddcd88e7..54d3.js gzip | 4.68 kB | 4.68 kB | ✓ |
| commons.HASH.js gzip | 4.06 kB | 4.06 kB | ✓ |
| de003c3a9d30..4cf7.js gzip | 13.7 kB | 13.7 kB | ✓ |
| framework.HASH.js gzip | 39.1 kB | 39.1 kB | ✓ |
| Overall change | 67.5 kB | 67.5 kB | ✓ |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons) Modern</strong></summary>

|  | zeit/next.js canary  | lfades/next.js docs-update-pathname | Change |
| - | - | - | - |
| main-HASH.module.js gzip | 4.19 kB | 4.19 kB | ✓ |
| webpack-HASH..dule.js gzip | 746 B | 746 B | ✓ |
| 4952ddcd88e7..dule.js gzip | 5.56 kB | 5.56 kB | ✓ |
| de003c3a9d30..dule.js gzip | 12.5 kB | 12.5 kB | ✓ |
| framework.HA..dule.js gzip | 39.1 kB | 39.1 kB | ✓ |
| Overall change | 62.1 kB | 62.1 kB | ✓ |

</details>

<details>
<summary><strong>Legacy Client Bundles (polyfills)</strong></summary>

|  | zeit/next.js canary  | lfades/next.js docs-update-pathname | Change |
| - | - | - | - |
| polyfills-HASH.js gzip | 4.76 kB | 4.76 kB | ✓ |
| Overall change | 4.76 kB | 4.76 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages</strong></summary>

|  | zeit/next.js canary  | lfades/next.js docs-update-pathname | Change |
| - | - | - | - |
| _app.js gzip | 1.33 kB | 1.33 kB | ✓ |
| _error.js gzip | 4.07 kB | 4.07 kB | ✓ |
| hooks.js gzip | 779 B | 779 B | ✓ |
| index.js gzip | 222 B | 222 B | ✓ |
| link.js gzip | 2.9 kB | 2.9 kB | ✓ |
| routerDirect.js gzip | 283 B | 283 B | ✓ |
| withRouter.js gzip | 282 B | 282 B | ✓ |
| Overall change | 9.87 kB | 9.87 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages Modern</strong></summary>

|  | zeit/next.js canary  | lfades/next.js docs-update-pathname | Change |
| - | - | - | - |
| _app.module.js gzip | 757 B | 757 B | ✓ |
| _error.module.js gzip | 3.06 kB | 3.06 kB | ✓ |
| hooks.module.js gzip | 371 B | 371 B | ✓ |
| index.module.js gzip | 212 B | 212 B | ✓ |
| link.module.js gzip | 2.47 kB | 2.47 kB | ✓ |
| routerDirect..dule.js gzip | 273 B | 273 B | ✓ |
| withRouter.m..dule.js gzip | 272 B | 272 B | ✓ |
| Overall change | 7.41 kB | 7.41 kB | ✓ |

</details>

<details>
<summary><strong>Client Build Manifests</strong></summary>

|  | zeit/next.js canary  | lfades/next.js docs-update-pathname | Change |
| - | - | - | - |
| _buildManifest.js gzip | 61 B | 61 B | ✓ |
| _buildManife..dule.js gzip | 61 B | 61 B | ✓ |
| Overall change | 122 B | 122 B | ✓ |

</details>

<details>
<summary><strong>Serverless bundles</strong></summary>

|  | zeit/next.js canary  | lfades/next.js docs-update-pathname | Change |
| - | - | - | - |
| _error.js gzip | 77.7 kB | 77.7 kB | ✓ |
| hooks.html gzip | 1.05 kB | 1.05 kB | ✓ |
| index.js gzip | 78 kB | 78 kB | ✓ |
| link.js gzip | 80.3 kB | 80.3 kB | ✓ |
| routerDirect.js gzip | 78 kB | 78 kB | ✓ |
| withRouter.js gzip | 78.1 kB | 78.1 kB | ✓ |
| Overall change | 393 kB | 393 kB | ✓ |

</details>

</details>


Commit: 4987cfb653e6154883b2608395e1c5f100ffb79a

**Comment by ijjk** [comment]
# Stats from current PR

<details>
<summary><strong>Default Server Mode</strong></summary>

<br/>

<details>
<summary><strong>General</strong> Overall increase ⚠️</summary>

|  | zeit/next.js canary  | lfades/next.js docs-update-pathname | Change |
| - | - | - | - |
| buildDuration | 12.5s | 12.7s | ⚠️ +123ms |
| nodeModulesSize | 48.9 MB | 48.9 MB | ⚠️ +10 B |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons)</strong></summary>

|  | zeit/next.js canary  | lfades/next.js docs-update-pathname | Change |
| - | - | - | - |
| main-HASH.js gzip | 5.13 kB | 5.13 kB | ✓ |
| webpack-HASH.js gzip | 746 B | 746 B | ✓ |
| 4952ddcd88e7..54d3.js gzip | 4.68 kB | 4.68 kB | ✓ |
| commons.HASH.js gzip | 4.06 kB | 4.06 kB | ✓ |
| de003c3a9d30..4cf7.js gzip | 13.7 kB | 13.7 kB | ✓ |
| framework.HASH.js gzip | 39.1 kB | 39.1 kB | ✓ |
| Overall change | 67.5 kB | 67.5 kB | ✓ |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons) Modern</strong></summary>

|  | zeit/next.js canary  | lfades/next.js docs-update-pathname | Change |
| - | - | - | - |
| main-HASH.module.js gzip | 4.19 kB | 4.19 kB | ✓ |
| webpack-HASH..dule.js gzip | 746 B | 746 B | ✓ |
| 4952ddcd88e7..dule.js gzip | 5.56 kB | 5.56 kB | ✓ |
| de003c3a9d30..dule.js gzip | 12.5 kB | 12.5 kB | ✓ |
| framework.HA..dule.js gzip | 39.1 kB | 39.1 kB | ✓ |
| Overall change | 62.1 kB | 62.1 kB | ✓ |

</details>

<details>
<summary><strong>Legacy Client Bundles (polyfills)</strong></summary>

|  | zeit/next.js canary  | lfades/next.js docs-update-pathname | Change |
| - | - | - | - |
| polyfills-HASH.js gzip | 4.76 kB | 4.76 kB | ✓ |
| Overall change | 4.76 kB | 4.76 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages</strong></summary>

|  | zeit/next.js canary  | lfades/next.js docs-update-pathname | Change |
| - | - | - | - |
| _app.js gzip | 1.33 kB | 1.33 kB | ✓ |
| _error.js gzip | 4.07 kB | 4.07 kB | ✓ |
| hooks.js gzip | 779 B | 779 B | ✓ |
| index.js gzip | 222 B | 222 B | ✓ |
| link.js gzip | 2.9 kB | 2.9 kB | ✓ |
| routerDirect.js gzip | 283 B | 283 B | ✓ |
| withRouter.js gzip | 282 B | 282 B | ✓ |
| Overall change | 9.87 kB | 9.87 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages Modern</strong></summary>

|  | zeit/next.js canary  | lfades/next.js docs-update-pathname | Change |
| - | - | - | - |
| _app.module.js gzip | 757 B | 757 B | ✓ |
| _error.module.js gzip | 3.06 kB | 3.06 kB | ✓ |
| hooks.module.js gzip | 371 B | 371 B | ✓ |
| index.module.js gzip | 212 B | 212 B | ✓ |
| link.module.js gzip | 2.47 kB | 2.47 kB | ✓ |
| routerDirect..dule.js gzip | 273 B | 273 B | ✓ |
| withRouter.m..dule.js gzip | 272 B | 272 B | ✓ |
| Overall change | 7.41 kB | 7.41 kB | ✓ |

</details>

<details>
<summary><strong>Client Build Manifests</strong></summary>

|  | zeit/next.js canary  | lfades/next.js docs-update-pathname | Change |
| - | - | - | - |
| _buildManifest.js gzip | 61 B | 61 B | ✓ |
| _buildManife..dule.js gzip | 61 B | 61 B | ✓ |
| Overall change | 122 B | 122 B | ✓ |

</details>

<details>
<summary><strong>Rendered Page Sizes</strong></summary>

|  | zeit/next.js canary  | lfades/next.js docs-update-pathname | Change |
| - | - | - | - |
| index.html gzip | 1.02 kB | 1.02 kB | ✓ |
| link.html gzip | 1.03 kB | 1.03 kB | ✓ |
| withRouter.html gzip | 1.02 kB | 1.02 kB | ✓ |
| Overall change | 3.07 kB | 3.07 kB | ✓ |

</details>

</details>
<hr/>
<details>
<summary><strong>Serverless Mode</strong></summary>

<br/>

<details>
<summary><strong>General</strong> Overall increase ⚠️</summary>

|  | zeit/next.js canary  | lfades/next.js docs-update-pathname | Change |
| - | - | - | - |
| buildDuration | 12.8s | 12.9s | ⚠️ +65ms |
| nodeModulesSize | 48.9 MB | 48.9 MB | ⚠️ +10 B |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons)</strong></summary>

|  | zeit/next.js canary  | lfades/next.js docs-update-pathname | Change |
| - | - | - | - |
| main-HASH.js gzip | 5.13 kB | 5.13 kB | ✓ |
| webpack-HASH.js gzip | 746 B | 746 B | ✓ |
| 4952ddcd88e7..54d3.js gzip | 4.68 kB | 4.68 kB | ✓ |
| commons.HASH.js gzip | 4.06 kB | 4.06 kB | ✓ |
| de003c3a9d30..4cf7.js gzip | 13.7 kB | 13.7 kB | ✓ |
| framework.HASH.js gzip | 39.1 kB | 39.1 kB | ✓ |
| Overall change | 67.5 kB | 67.5 kB | ✓ |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons) Modern</strong></summary>

|  | zeit/next.js canary  | lfades/next.js docs-update-pathname | Change |
| - | - | - | - |
| main-HASH.module.js gzip | 4.19 kB | 4.19 kB | ✓ |
| webpack-HASH..dule.js gzip | 746 B | 746 B | ✓ |
| 4952ddcd88e7..dule.js gzip | 5.56 kB | 5.56 kB | ✓ |
| de003c3a9d30..dule.js gzip | 12.5 kB | 12.5 kB | ✓ |
| framework.HA..dule.js gzip | 39.1 kB | 39.1 kB | ✓ |
| Overall change | 62.1 kB | 62.1 kB | ✓ |

</details>

<details>
<summary><strong>Legacy Client Bundles (polyfills)</strong></summary>

|  | zeit/next.js canary  | lfades/next.js docs-update-pathname | Change |
| - | - | - | - |
| polyfills-HASH.js gzip | 4.76 kB | 4.76 kB | ✓ |
| Overall change | 4.76 kB | 4.76 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages</strong></summary>

|  | zeit/next.js canary  | lfades/next.js docs-update-pathname | Change |
| - | - | - | - |
| _app.js gzip | 1.33 kB | 1.33 kB | ✓ |
| _error.js gzip | 4.07 kB | 4.07 kB | ✓ |
| hooks.js gzip | 779 B | 779 B | ✓ |
| index.js gzip | 222 B | 222 B | ✓ |
| link.js gzip | 2.9 kB | 2.9 kB | ✓ |
| routerDirect.js gzip | 283 B | 283 B | ✓ |
| withRouter.js gzip | 282 B | 282 B | ✓ |
| Overall change | 9.87 kB | 9.87 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages Modern</strong></summary>

|  | zeit/next.js canary  | lfades/next.js docs-update-pathname | Change |
| - | - | - | - |
| _app.module.js gzip | 757 B | 757 B | ✓ |
| _error.module.js gzip | 3.06 kB | 3.06 kB | ✓ |
| hooks.module.js gzip | 371 B | 371 B | ✓ |
| index.module.js gzip | 212 B | 212 B | ✓ |
| link.module.js gzip | 2.47 kB | 2.47 kB | ✓ |
| routerDirect..dule.js gzip | 273 B | 273 B | ✓ |
| withRouter.m..dule.js gzip | 272 B | 272 B | ✓ |
| Overall change | 7.41 kB | 7.41 kB | ✓ |

</details>

<details>
<summary><strong>Client Build Manifests</strong></summary>

|  | zeit/next.js canary  | lfades/next.js docs-update-pathname | Change |
| - | - | - | - |
| _buildManifest.js gzip | 61 B | 61 B | ✓ |
| _buildManife..dule.js gzip | 61 B | 61 B | ✓ |
| Overall change | 122 B | 122 B | ✓ |

</details>

<details>
<summary><strong>Serverless bundles</strong></summary>

|  | zeit/next.js canary  | lfades/next.js docs-update-pathname | Change |
| - | - | - | - |
| _error.js gzip | 77.7 kB | 77.7 kB | ✓ |
| hooks.html gzip | 1.05 kB | 1.05 kB | ✓ |
| index.js gzip | 78 kB | 78 kB | ✓ |
| link.js gzip | 80.3 kB | 80.3 kB | ✓ |
| routerDirect.js gzip | 78 kB | 78 kB | ✓ |
| withRouter.js gzip | 78.1 kB | 78.1 kB | ✓ |
| Overall change | 393 kB | 393 kB | ✓ |

</details>

</details>


Commit: 148e4ab76589123a4cdf3c5da0ceb5ddc97b9def

**Review by timneutkens — APPROVED** [review]

**Pull request merged**